src/Protocol/Protocol_1_9.cpp

```diff
@@ -2665,8 +2665,11 @@ void cProtocol_1_9_0::HandlePacketUseEntity(cByteBuffer & a_ByteBuffer)
 	{
 		case 0:
 		{
-			HANDLE_READ(a_ByteBuffer, ReadVarInt, UInt32, Hand)
-			m_Client->HandleUseEntity(EntityID, false);
+			HANDLE_READ(a_ByteBuffer, ReadVarInt, UInt32, Hand);
+			if (Hand == 0)  // TODO: implement handle off-hand (1) actions
```

Comment:
I'm not sure if this comment makes it completely clear why we're ignoring it.  Maybe something like

```c++
// TODO: implement handling of off-hand actions; ignore them for now to avoid processing actions twice
```

would be better.